src/BuildToolsUnitTests/BuildToolsUnitTests.csproj

```diff
@@ -26,11 +26,19 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Include="AOT\Schemas\Enums.cs" />
     <None Update="AOT/Schemas/**/*.proto" CopyToOutputDirectory="PreserveNewest" />
     <None Update="AOT/Schemas/**/*.json" CopyToOutputDirectory="PreserveNewest" DependentUpon="%(Filename).proto" />
     <None Update="AOT/Schemas/**/*.pjson" CopyToOutputDirectory="PreserveNewest" DependentUpon="%(Filename).proto" />
 
     <Compile Remove="AOT/Schemas/**/*.cs" />
     <None Include="AOT/Schemas/**/*.cs" CopyToOutputDirectory="PreserveNewest" DependentUpon="%(Filename).proto" />
+    <None Update="AOT\Schemas\Basic.cs.json">
```

Comment:
the *intent* was to avoid needing to saturate the csproj with per-test elements here; if we *can* make it work just using the wildcards, that would IMO be preferable; also, I suspect we don't actually want to compile the test files (`Enum.cs` etc) - unless we're doing that temporarily just to see what is broken

Reply:
as you have shown, updated other files using #define. Removed all .cs.json files